d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
```

Comment:
Not that I am aware of.  Any issues related to something reset stylesheet ought to be dealt with in a CSS style for the affected element, so I don't think this generic mixin should be setting properties to deal with such an issue.  

I think we should just inherit the color in this case, and if we run into a scenario where we need to be explicit in the mixin then we reevaluate. I kinda of want to update the small-text mixin too, but we'd need to due a little investigating to make sure there would not be any regression.  Given that, I'd probably not do that as part of this change.